packages/firebase_app_check/firebase_app_check/example/lib/main.dart

```diff
@@ -17,6 +18,7 @@ Future<void> main() async {
       // Your personal reCaptcha public key goes here:
       .activate(
     webRecaptchaSiteKey: kWebRecaptchaSiteKey,
+    androidDebugProvider: true,
```

Comment:
Will it work if called twice?

Reply:
Yes, it's fine. Just prints the same debug token into the console.